python/pip_install/pip_requirements.bzl

```diff
@@ -0,0 +1,68 @@
+"Rules to verify and update pip-compile locked requirements.txt"
+
+load("//python:defs.bzl", "py_binary", "py_test")
+load("//python/pip_install:repositories.bzl", "requirement")
+
+def pip_requirements(
```

Comment:
With this name it seems easy enough to confuse its purpose with `pip_install`. I'd favor a more specific name like `compile_pip_requirements`.

Reply:
`compile_pip_requirements` works for me. Also changed the load location to just "requirements.bzl" since "pip_" is redundant with the package name, and rules_python doesn't have a convention about filenames matching the rule/macro name